templates/default/github_comment.config.xml.erb

```diff
@@ -0,0 +1,64 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project>
+  <actions/>
+  <description>Send a comment to a specified Github PR</description>
+  <keepDependencies>false</keepDependencies>
+  <properties>
+    <com.dabsquared.gitlabjenkins.connection.GitLabConnectionProperty plugin="gitlab-plugin@
+1.4.4">
+      <gitLabConnection>https://git.osuosl.org</gitLabConnection>
+    </com.dabsquared.gitlabjenkins.connection.GitLabConnectionProperty>
```

Comment:
I'm assuming we can go ahead and remove the gitlab reference since this is for github?